Classes/IRC/IRCClient.m

```diff
@@ -2593,14 +2593,15 @@ - (void)receiveDCCSend:(IRCMessage*)m fileName:(NSString*)fileName address:(NSSt
 
     if ([Preferences dccAction] != DCC_IGNORE) {
         if (port > 0 && size > 0) {
-            NSString* path = [@"~/Downloads" stringByExpandingTildeInPath];
+            NSString* path = [Preferences getDownloadsPath];
             NSFileManager* fm = [NSFileManager defaultManager];
             BOOL isDir = NO;
             if ([fm fileExistsAtPath:path isDirectory:&isDir]) {
-                path = @"~/Downloads";
+                ;
```

Comment:
You can remove this bit if you switch the `if` to become negative, and then you'll only need the part which you currently have in `else`